File: AkitaModel/Results/Product.cs
namespace Skyware.Lis.AkitaModel.Results;

/// <summary>
/// Represents a orderable/payable product in Sale.
/// </summary>
public class Product : ItemBase
{

    /// <summary>
    /// Price of the product, payable by the patient.
    /// </summary>
    public decimal Price { get; set; }

}

File: AkitaMotelTS/src/Results/Product.ts
/**
 * Skyware.Lis.AkitaModel.Results
 */

import { ItemBase } from './ItemBase';

/**
 * Represents a orderable/payable product in Sale.
 */
export class Product extends ItemBase {
  /**
   * Price of the product, payable by the patient.
   */
  price!: number;
}
